fe/fe-core/src/main/java/com/starrocks/persist/Storage.java

```diff
@@ -210,6 +210,23 @@ public void writeFrontendRoleAndNodeName(FrontendNodeType role, String nameNode)
         }
     }
 
+    public void writeFeStartFeHostTypeAndHost(String hostType, String host) throws IOException {
```

Comment:
This function writes all of the 4 properties, it dose not match the function name.
You can just set the role and nodeName to the properties, and there is no need to use writeFrontendRoleAndNodeName.